nbs/core.ipynb

```diff
@@ -61,7 +61,8 @@
     "from typing import Any, List, Optional\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #34.            <code>verbose</code>: bool, Wether print progress bar. Only used when <code>n_jobs=1</code>.

</pre><p>Suggestion: "Prints TQDM progress bar when n_jobs=1" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/280/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
fixed in [c2e915c](https://github.com/Nixtla/statsforecast/pull/280/commits/c2e915c18286524dfcf9b42b179a228d5ce4bbc2)